server/api/db/sqldb/db.py

```diff
@@ -5907,6 +5936,14 @@ def delete_table_records_by_name(
             table_name=sanitized_table_name,
         )
 
+    def _delete_project_summary(
+        self,
+        session: Session,
+        name: str,
+    ):
+        logger.debug("Deleting project summary from DB", name=name)
+        self._delete(session, ProjectSummary, project=name)
```

Comment:
move this function up, below the rest of the public `project_summary` methods.
The order here is not exactly public then private, we strive to keep the resource-related methods in the same area.